homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,296 @@
+"""Support for Lektrico charging station sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from lektricowifi import Device
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_FRIENDLY_NAME,
+    PERCENTAGE,
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import LektricoEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class LektricoSensorEntityDescription(SensorEntityDescription):
+    """A class that describes the Lektrico sensor entities."""
+
+    value_fn: Callable[[Any], float | str | int]
+
+
+SENSORS_FOR_CHARGERS: tuple[LektricoSensorEntityDescription, ...] = (
+    LektricoSensorEntityDescription(
+        key="state",
+        translation_key="state",
+        value_fn=lambda data: str(data.charger_state),
+    ),
```

Comment:
What can the state be?

Reply:
The state can be:  
"Available", "Connected", "Connected,NeedAuth", "Paused", "Charging", "Error", "Updating firmware".

Reply:
Then this sensor should be an enum sensor where the values are in snake_case ("available", "need_auth", "updating_firmware") and the translated values in the `strings.json`

Reply:
I made the changes, thanks.